src/content/guides/getting-started.md

```diff
@@ -279,6 +311,6 @@ webpack-demo
 |- /node_modules
 ```
 
-T> If you're using npm 5, you'll probably also see a `package-lock.json` file in your directory.
+T> If you're using npm 5, you'll probably also see a `package-lock.json` file in your directory as well.
```

Comment:
Isn't this a redundancy: "also" and "as well" at the end?

Reply:
Nice catch!